doc/_templates/index.html

```diff
@@ -178,8 +178,10 @@
   <h1>Open source</h1>
 
 <p>
-Please consider <a href="http://sourceforge.net/project/project_donations.php?group_id=80706">donating</a>
-to support matplotlib development or to the <a href="http://numfocus.org/johnhunter/">John Hunter Memorial Fund</a>.
+Please consider <a href="http://numfocus.org/projects/">donating
+through Numfocus</a> to support matplotlib development or to
```

Comment:
I don't think this link to Numfocus is the best we can do -- we should try to get something more direct.